webapp/loggedin/modules/users/userList/userListView.js

```diff
@@ -77,7 +77,6 @@ const UsersListView = ({ canInvite, user, surveyInfo, history }) => {
       module={'users'}
       className="users-list"
       gridTemplateColumns={'35px repeat(3, 1fr) 10rem 50px'}
-      headerLeftComponent={UsersHeaderLeft}
       rowHeaderComponent={UsersRowHeader}
```

Comment:
i would leave the UsersHeaderLeft, make sure invitation is working

Reply:
ok